application/view/omeka/admin/media/browse.phtml

```diff
@@ -31,19 +33,56 @@ $sortHeadings = [
     <?php echo $this->sortSelector($sortHeadings); ?>
 </div>
 
+<form method="post" id="batch-form" class="disable-unsaved-warning">
+
+<div id="page-actions">
```

Comment:
I believe this div should be `class="batch-inputs"` not `id="page-actions"`

Reply:
The initial pr was six month ago... Some water has flowed under the bridges. I fixed it with the new commits.